Jenkinsfile.nightly

```diff
@@ -9,8 +9,10 @@ pipeline {
         jdk 'kie-jdk1.8'
     }
     parameters {
-        string(defaultValue: 'master', description: 'The default branch', name: 'DEFAULT_BRANCH')
         string(description: 'The deployment URL', name: 'KIE_GROUP_DEPLOYMENT_REPO_URL')
+        booleanParam(description: 'Skip Tests? True as default', name: 'SKIP_TESTS', defaultValue: true)
+        string(description: 'The UMB message version', name: 'UMB_VERSION', defaultValue: 'master')
```

Comment:
Should the defaultValue `master` be here? I know it is overridden when the pipeline is actually run, but maybe it would be safer to put here a default for `7.33.x`?

Reply:
In case we say 7.33.x we will have to update this file each time the default branch is deprecated. I propose to remove the default in case master is not right.

Reply:
@winklerm I just noticed this is the backported one. So fixing it.